doc/userguide/conf.py

```diff
@@ -335,3 +335,7 @@ def setup(app):
     "sysconfdir": os.getenv("sysconfdir", "/etc"),
     "localstatedir": os.getenv("localstatedir", "/var"),
 }
+
+
+# for using reference numbers
+numfig = True
```

Comment:
this should go in a separate commit

Reply:
Should only this change (regarding conf.py change) be in a separate commit or all the docs changes?

Reply:
this updated general doc config, so that should be separate from any dpdk specific thing

Reply:
ok, the change has been separated.